src/test.cpp

```diff
@@ -0,0 +1,382 @@
+            
+            if (inChar == '\n') {
+                if (inString == "fin"){
+                    return;
+                } else if (inString == "quiet"){
+                    isPrint = false;
+                } else if (inString == "loud"){
+                    isPrint = true;
+                } else{
+                    // Start a flow
+                    isPrint = true;
+                    setPoint=inString.toInt(); //Unable to Abstract because of this setPoint Value
+                    flowStart = millis();
+                    printFreq = 50;
+                }
+                inString = "";
+            } else {
+                inString += (char)inChar;
+            }
+            
+        }
+    }
+    }
+
+
+    void angleSweep(long startAngle, long endAngle, unsigned long flowDuration, long extraTime) {
+        PID test = PID(11.5, 1.5e-6, 0.21e6, 130, false);
+        long setPoint = 130;
+        unsigned long lastPrint = 0;
+        long angle;
+        unsigned long flowStart = millis(); // in millis
+        bool isPrint = true;
+        unsigned int printFreq = 20;
+        while (true) {
+            speed = test.update(encoder.read());
+            utility::runMotor(speed);
+            if (isPrint && (millis()-lastPrint > printFreq)){ 
+                #ifndef USE_DASHBOARD
+                Serial.println(String(millis()) + "\t" + String(speed)+"\t"+String(angle)+"\t"+String(setPoint) + "\t" + String(utility::voltageToHighPressure(analogRead(HP_PT))) + "\t" + String(utility::voltageToPressure(analogRead(LP_PT))));
+                #else
+                Comms::Packet packet = {.id = 1};
+                // Comms::packetAddFloat(&packet, sin(t2/1e6));
+                Comms::packetAddFloat(&packet, float(setPoint));
+                Comms::packetAddFloat(&packet, 0);
+                Comms::packetAddFloat(&packet, float(speed));
+                Comms::packetAddFloat(&packet, angle);
+                Comms::packetAddFloat(&packet, utility::voltageToHighPressure(analogRead(HP_PT)));
+                Comms::packetAddFloat(&packet, utility::voltageToPressure(analogRead(LP_PT)));
+                Comms::packetAddFloat(&packet, 0);
+                Comms::packetAddFloat(&packet, 0);
+                Comms::packetAddFloat(&packet, 0);
+                Comms::emitPacket(&packet);
+                #endif
+            
+                lastPrint = millis();
+            }
+
+            if (millis()-flowStart > flowDuration) { // flow has ended, close valves
+                setPoint = 0;
+                printFreq = 5000;
+            } else {
+                float prog = float(millis()-flowStart)/float(flowDuration);
+                setPoint = prog * endAngle + (1-prog) * startAngle;
+            }
+
+            if (millis()-flowStart > flowDuration + extraTime) { // flow has ended, close valves
+                return;
+            }
+        }
+    }
+
+    boolean pressureize_tank() {
```

Comment:
spelling: pressurize_tank

Also, this isn't a test function, it's used to make sure the LP tank starts the control sequence at nearly the right pressure (to prevent pressure from overshooting the setpoint) - i'd suggest moving to utility